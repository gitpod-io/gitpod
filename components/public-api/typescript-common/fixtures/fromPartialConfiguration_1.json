{
  "id": "123",
  "name": "My Config"
}
