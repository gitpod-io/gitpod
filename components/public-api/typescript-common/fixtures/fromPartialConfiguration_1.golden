{
    "result": {
        "id": "123",
        "settings": {},
        "name": "My Config"
    },
    "err": ""
}
